documentation/modules/ROOT/pages/operations/debezium-ui.adoc

```diff
@@ -0,0 +1,69 @@
+[id="debezium-ui"]
+= {prodname} UI
+
+:linkattrs:
+:icons: font
+:toc:
+:toclevels: 3
+:toc-placement: macro
+
+toc::[]
+
+The {prodname} UI allows users to set up and operate connectors more easily.  A list of all connectors can be viewed along with their status, etc.  The Create Connector wizard allows the user to configure connectors, guiding and validating the property entries along the way.
+
+== Design Considerations
+
+The UI is implemented as a https://quarkus.io/[Quarkus]-based web application.  The backend is configured with the URL(s) of one or more Kafka Connect clusters and provides a REST interface for the frontend.  The frontend user interface uses https://reactjs.org/[ReactJS] as the primary technology, utilizing https://www.patternfly.org/v4/[Patternfly] react components and design patterns.
+
+== Installation and Configuration
+
+==== debezium-ui docker image
+
+The debezium-ui https://hub.docker.com/r/debezium/debezium-ui[docker image] is available for running the UI.  To start the UI and connect to an existing Kafka Connect instance:
+
+----
+$ docker run -it --rm --name debezium-ui -p 8080:8080 -e KAFKA_CONNECT_URI=http://connect:8083 -e PROXY=true --link connect:connect debezium/debezium-ui:1.5
```

Comment:
1) What is that `PROXY` env var?

2) I think `--link` is only needed in certain environments and should be removed, but can you add a paragraph on that the UI requires a working network connection to a Kafka Connect instance (via running on the same docker network or via `--link`, but link is considered as deprecated, so I would maybe not mention it at all)

3) We should name the other config parameters as well (see the `@ConfigProperty` annotated fields in the backend code and KafkaConnectClientFactory#PROPERTY_KAFKA_CONNECT_URI)

Reply:
yeah, now I see that `--link` is deprecated we can remove it by adding a note to run the UI on the same docker network.

Reply:
I think `PROXY`  is to set CORS configuration in Kafka Connect but I am not sure if its required here.

Reply:
@rk3rn3r - For (3) Are you saying we should make changes on the backend to name additional parameters?  Or utilize existing params in the command?

Reply:
I updated the PR - changed the docker command to remove the --link and PROXY per comments 1 & 2.  And added note about the docker network.